src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -78,6 +78,16 @@
                 }
             }";
 
+const string SimpleVecOpTest_ValidationLogicForCndSelForNarrowing = @"for (var i = 0; i < Op1ElementCount; i++)
```

Comment:
Nit: Can we rename this to `SimpleVecOpTest_ValidationLogicForCndSelNarrowing`, and do the same for `SimpleVecOpTest_ValidationLogicForNarrowing` so it's `SimpleVecOpTest_ValidationLogicNarrowing`? We seem to use the `Narrowing` suffix on its own elsewhere in the file, so this seems to be a bit more consistent.